tests/Integration/bootstrap.php

```diff
@@ -39,4 +39,6 @@
     define('PHPUNIT_COMPOSER_INSTALL', dirname(__DIR__, 4) . '/vendor/autoload.php');
 }
 
-require_once dirname(__DIR__, 2) . '/vendor/autoload.php';
+if (file_exists(dirname(__DIR__, 2) . '/vendor/autoload.php')) {
+    require_once dirname(__DIR__, 2) . '/vendor/autoload.php';
+}
```

Comment:
Ahh after so many years I learn about the difference between include and require 😅 Thanks @mflasquin 
However after a little digging, it seems like include doesn't trigger an error indeed but it triggers a warning instead, so we should probably still check the existence of the file to prevent polluting the logs with unwanted warnings

Reply:
Also besides the file checking I'm not sure but if an error occurs in the file itself while being loaded I'm concerned include may ignore it and simply display a warning But it's better to have an actual error when the file is indeed present IMHO

Reply:
thank you for the suggestion :) I review my copy